### PR TITLE
Added VolumeWeightTween for SRP post-processing volume blending

### PR DESCRIPTION
**Pull request by stefankohl-dev**
Added VolumeWeightTween for SRP post-processing volume blending

### CODE REVIEW

**Comment by jeffreylanters** [comment]
Awesome Stefan, thanks for your contribution!

File: definitions.asmdef [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
     "includePlatforms": [],
     "excludePlatforms": [],
     "allowUnsafeCode": false,
     "overrideReferences": false,
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": []
-}
+    "versionDefines": [
```
Comment (jeffreylanters) [inline definitions.asmdef:14]: What exactly does this do? I never used references to other dependencies. Does this mean `com.unity.render-pipelines.core` becomes a required dependency of this module?

Comment (jeffreylanters) [reply definitions.asmdef:14]: Ah -- I think I understand. When this module is available, the SRP_AVAILABLE assembly definition will be set, right?

File: Tests/TweenTests.cs [hunk 2]
```diff
@@ -91,6 +91,8 @@ public class TweenTests : MonoBehaviour {
         this.target.TweenSpriteRendererColor (Color.white, 1).SetFrom (Color.red);
       if (GUILayout.Button ("TextMeshAlpha"))
         this.target.TweenTextMeshAlpha (1, 1).SetFrom (0);
+      if (GUILayout.Button ("VolumeWeight"))
+        this.target.TweenVolumeWeight (1, 1).SetFrom (0);
```
Comment (jeffreylanters) [inline Tests/TweenTests.cs:95]: I haven't tested it yet, but I think this could error when the assembly definition `SRP_AVAILABLE` is not defined.

**Review by jeffreylanters — CHANGES_REQUESTED** [review]

File: definitions.asmdef (hunk 1 above)
Comment (stefankohl-dev) [reply definitions.asmdef:14]: Yes, the preprocessor symbol will be generated only if the SRP Core package is available. If not, then the tween class will not be available.

Sorry, I forgot to update the tests. The SRP_AVAILABLE condition also needs to be checked against the test case:

`#if SRP_AVAILABLE
      if (GUILayout.Button ("VolumeWeight"))
        this.target.TweenVolumeWeight (1, 1).SetFrom (0);
#endif`

Comment (jeffreylanters) [reply definitions.asmdef:14]: Thanks for clearing that up! I'll release you contribution. 🚀

**Pull request merged**